src/NServiceBus.AcceptanceTests/Sagas/When_message_has_a_saga_id.cs

```diff
@@ -66,7 +66,7 @@ public Task Handle(MessageWithSagaId message, IMessageHandlerContext context)
                     return Task.FromResult(0);
                 }
 
-                public Task Handle(object message, IMessageProcessingContext context)
+                public Task Handle(object message, IMessageProcessingContext context, Type sagaType)
```

Comment:
Nitpick: Would it be more logical to pass the saga type as the second argument since the message and the saga type is the "input" and the processing context contains the "actions" that the user can take? (Could we even consider a SagaHandlerContext perhaps?)

Reply:
I think that would be better. Pretty simple to do. I'll do that hopefully next week.